consensus/safety-rules/src/safety_rules.rs

```diff
@@ -219,6 +219,16 @@ impl SafetyRules {
     // Internal functions mapped to the public interface to enable exhaustive logging and metrics
 
     fn guarded_consensus_state(&mut self) -> Result<ConsensusState, Error> {
+        let waypoint = self.persistent_storage.waypoint()?;
+        let safety_data = self.persistent_storage.safety_data()?;
+
+        info!(SafetyLogSchema::new(LogEntry::State, LogEvent::Update)
```

Comment:
this is only called together with initialize, maybe log it there?

Reply:
I also call it in when the remote service starts up. Considering this can be called arbitrarily whereas initialize cannot, it seemed safer to put here.